components/tools/OmeroPy/test/integration/test_isession.py

```diff
@@ -288,7 +288,11 @@ def testManageMySessions(self):
         finally:
             c1.__del__()
 
-    def testSessionWithIP(self):
+    @pytest.mark.parametrize("client_ip", [
+        "127.0.0.1",
+        "2400:cb00:2048:1::6814:55",
+        "1234:5678:1234:5678:1234:5678:121.212.121.212"])
```

Comment:
for the curious reviewer, this imitates a IPv4-mapped IPv6 address from a dual-stack node